parquet-generator/src/main/java/org/apache/parquet/encoding/bitpacking/ByteBasedBitPackingGenerator.java

```diff
@@ -256,4 +254,8 @@ private static String align(int value, int digits) {
     result.append(valueString);
     return result.toString();
   }
+  
+  private static int paddedByteCountFromBits(int bitLength) {
```

Comment:
I will add a comment about ByteUtils. Not sure it is worth fixing though as the method is one-liner, is not parquet specific and easy to understand.